### PR TITLE
feature/testGroupsList

### PR DESCRIPTION
**Pull request by BassTester**
Groups list and members with mocked responses from server

### CODE REVIEW

File: app/src/main/java/jkartkowka/jkartkwkamobile/MainMenuWireframe.java [hunk 1]
```diff
@@ -1,5 +1,7 @@
 package jkartkowka.jkartkwkamobile;
 
+import android.provider.ContactsContract;
```
Comment (marian-sobczyk) [inline app/src/main/java/jkartkowka/jkartkwkamobile/MainMenuWireframe.java:3]: Co to za paskudztwo?

Comment (BassTester) [inline app/src/main/java/jkartkowka/jkartkwkamobile/MainMenuWireframe.java:3]: Cudo dodane przez android studio przypadkowo

Comment (marian-sobczyk) [inline app/src/main/java/jkartkowka/jkartkwkamobile/MainMenuWireframe.java:3]: To wywal ;)

2015-12-22 22:08 GMT+01:00 BassTester notifications@github.com:

Comment (marian-sobczyk) [inline app/src/main/java/jkartkowka/jkartkwkamobile/MainMenuWireframe.java:3]: To wywal :)

**Pull request closed without merging**